hydro_cli/src/core/gcp.rs

```diff
@@ -142,7 +146,14 @@ impl LaunchedHost for LaunchedComputeEngine {
         drop(created_file);
 
         let mut channel = session.channel_session().await?;
-        channel.exec(binary_path.to_str().unwrap()).await?;
+        let binary_path_string = binary_path.to_str().unwrap();
+        let args_string = args
+            .iter()
+            .map(|s| format!("\"{s}\""))
```

Comment:
MMMMMMMmmmmmmmmmm lets go ahead and replace `format!("\"{s}\"")` with `shell_escape::unix::escape(Cow::Owned(s))` https://docs.rs/shell-escape/0.1.5/shell_escape/unix/fn.escape.html

Reply:
oh amazing this is exactly what I was looking for